isacc_messaging/models/email.py

```diff
@@ -37,7 +41,7 @@ def send_email(recipient_emails: list, subject, text, html):
     try:
         with smtplib.SMTP_SSL(email_server, port, context=context) as server:
             server.login(sender_email, app_password)
-            server.sendmail(sender_email, recipient_emails, msg.as_string())
+            server.sendmail(sender_name, recipient_emails, msg.as_string())
```

Comment:
```suggestion
            server.sendmail(from_addr=sender_name, to_addrs=recipient_emails, msg.as_string())
```

Reply:
once you start naming kwargs, you must name all that follow or it'll raise:
```SyntaxError: positional argument follows keyword argument```